openbb_platform/providers/biztoc/openbb_biztoc/models/global_news.py

```diff
@@ -0,0 +1,164 @@
+"""Biztoc Globl News Fetcher."""
+from datetime import datetime, timedelta
+from typing import Any, Dict, List, Literal, Optional
+
+from openbb_biztoc.utils.helpers import get_all_tags, get_pages, get_sources
+from openbb_provider.abstract.fetcher import Fetcher
+from openbb_provider.standard_models.global_news import (
+    GlobalNewsData,
+    GlobalNewsQueryParams,
+)
+from openbb_provider.utils.helpers import make_request
+from pandas import to_datetime
+from pydantic import Field, field_validator
+
+
+class BiztocGlobalNewsQueryParams(GlobalNewsQueryParams):
+    """Biztoc Global News QueryParams."""
+
+    filter: Literal[
+        "crypto", "hot", "latest", "main", "media", "source", "tag"
+    ] = Field(default="latest", description="Filter by type of news.")
+    source: str = Field(
+        description="Filter by a specific publisher. Only valid when filter is set to source.",
+        default="bloomberg",
+    )
+    tag: Optional[str] = Field(
+        description="Tag, topic, to filter articles by. Only valid when filter is set to tag.",
+        default=None,
+    )
+    term: Optional[str] = Field(
+        description="Search term to filter articles by. This overrides all other filters.",
+        default=None,
+    )
+
+
+class BiztocGlobalNewsData(GlobalNewsData):
+    """Biztoc Global News Data."""
+
+    __alias_dict__ = {"date": "created", "text": "body", "site": "domain"}
+
+    images: Optional[Dict[str, str]] = Field(
+        description="Images for the article.", alias="img", default=None
+    )
+    favicon: Optional[str] = Field(
+        description="Icon image for the source of the article.", default=None
+    )
+    tags: Optional[List[str]] = Field(description="Tags for the article.", default=None)
+    id: Optional[str] = Field(description="Unique Article ID.", default=None)
+    score: Optional[float] = Field(
+        description="Search relevance score for the article.", default=None
+    )
+
+    @field_validator("date", "updated", mode="before", check_fields=False)
+    def date_validate(cls, v):  # pylint: disable=E0213
+        """Return formatted datetime."""
+        return to_datetime(v).strftime("%Y-%m-%d %H:%M:%S")
+
+
+class BiztocGlobalNewsFetcher(
+    Fetcher[
+        BiztocGlobalNewsQueryParams,
+        List[BiztocGlobalNewsData],
+    ]
+):
+    """Transform the query, extract and transform the data from the Biztoc endpoints."""
+
+    @staticmethod
+    def transform_query(params: Dict[str, Any]) -> BiztocGlobalNewsQueryParams:
+        """Transform the query."""
+        return BiztocGlobalNewsQueryParams(**params)
+
+    @staticmethod
+    def get_news(
```

Comment:
This function should be moved to `utils/helpers.py`

Reply:
moved